include/mbedtls/aes.h

```diff
@@ -1,9 +1,11 @@
 /**
  * \file aes.h
  *
- * \brief AES block cipher
+ * \brief   AES is a family of block ciphers that processes data in multiples
+ *          of block sizes (16 Bytes).
```

Comment:
Blocker: This comment is too generic - every block cipher processes data in blocks of fixed size.